ArduPlane/radio.cpp

```diff
@@ -211,7 +211,11 @@ void Plane::read_radio()
 
     control_failsafe();
 
-    if (g.throttle_nudge && channel_throttle->get_control_in() > 50 && geofence_stickmixing()) {
+    if (g.throttle_nudge && channel_throttle->get_control_in() > 50 
+        #if AC_FENCE == ENABLED
+        && fence_stickmixing()
```

Comment:
it's a bit hard to read having just part of the if within the #if so I'd probably instead have a local variable just above the if.

i.e. something like this:

    #if AC_FENCE == ENABLED
        const bool stickmixing = fence_stickmixing();
    #else
        const bool stickmixing = true;
    #end 
    if (g.throttle_nudge && channel_throttle->get_control_in() > 50 && stickmixing) {